projects/noloan2/app/src/androidTest/java/com/example/gal/noloan/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.gal.noloan;
```

Comment:
Do we have to keep this long folder structure?
`app/src/androidTest/java/com/example/gal/noloan/ExampleInstrumentedTest.java`
How about just
`app/androidTest/ExampleInstrumentedTest.java`
?